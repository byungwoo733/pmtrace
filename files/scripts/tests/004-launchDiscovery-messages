2016-05-10T08:01:35.304870Z [22642.191864919] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":22642.191,"PerfType":"Event","PerfGroup":"Input","key_code":773,"key_value":1} 
2016-05-10T08:01:35.486171Z [22642.373174753] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":22642.372,"PerfType":"Event","PerfGroup":"Input","key_code":773,"key_value":0} 
2016-05-10T08:01:35.508463Z [22642.395470961] user.info surface-manager-starfish [] qmlPerformance HOME_LAUNCHER_SHOWN {     "PerfGroup": "HomeScreenShown",     "PerfType": "LSM",     "TIME": 21,     "UNIT": "ms" }  
2016-05-10T08:01:37.556206Z [22644.443208379] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":22644.442,"PerfType":"Event","PerfGroup":"Input","key_code":272,"key_value":1} 
2016-05-10T08:01:37.716311Z [22644.603315962] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":22644.602,"PerfType":"Event","PerfGroup":"Input","key_code":272,"key_value":0} 
2016-05-10T08:01:37.745365Z [22644.632359212] user.info sam [] SAM APP_LAUNCH {"CLOCK":22644.632,"app_id":"com.webos.app.discovery","status":"start_prelaunching","PerfType":"AppLaunch","PerfGroup":"com.webos.app.discovery"} 
2016-05-10T08:01:37.758458Z [22644.645451879] user.info sam [] SAM APP_LAUNCH {"CLOCK":22644.645,"app_id":"com.webos.app.discovery","status":"start_memory_checking","PerfType":"AppLaunch","PerfGroup":"com.webos.app.discovery"} 
2016-05-10T08:01:37.760130Z [22644.647124379] user.info sam [] SAM APP_LAUNCH {"CLOCK":22644.646,"app_id":"com.webos.app.discovery","status":"start_launching","PerfType":"AppLaunch","PerfGroup":"com.webos.app.discovery"} 
2016-05-10T08:01:37.760784Z [22644.647778920] user.info WebAppMgr [] WAM APPLAUNCH_START {"CLOCK":22644.647,"PerfType":"AppLaunch","PerfGroup":"com.webos.app.discovery","APP_ID":"com.webos.app.discovery"} 
2016-05-10T08:01:37.765392Z [22644.652386462] user.info WebAppMgr [] WAM APP_ATTACHED_TO_CONTAINER {"CLOCK":22644.649,"PerfType":"AppLaunch","PerfGroup":"com.webos.app.discovery","APP_ID":"com.webos.app.discovery","PID":14242} 
2016-05-10T08:01:41.118166Z [22648.005160505] user.info WebAppMgr [] WAM APPLOADED {"CLOCK":22648.005,"PerfType":"AppLaunch","PerfGroup":"com.webos.app.discovery","APP_ID":"com.webos.app.discovery","URL":"https://kr.lgrecommends.lgappstv.com/discovery_2016/discovery.html?q=%7B%22returnValue%22%3Atrue%2C%22header%22%3A%22%22%2C%22HOST%22%3A%22KR.lgtvsdp.com%22%2C%22X-Device-Language%22%3A%22ko-KR%22%2C%22X-Device-FW-Version%22%3A%2200.00.00%22%2C%22X-Device- ... 3A%22HE_DTV_W16M_AFADATAA%22%2C%22X-Device-Eco-Info%22%3A%221%22%2C%22X-Device-Product%22%3A%22webOSTV+3.0%22%2C%22X-Device-Platform%22%3A%22W16M%22%2C%22X-Device-Publish-Flag%22%3A%22Y%22%2C%22X-Device-Sales-Model%22%3A%22WEBOS3%22%7D&page=store&fast=true","PID":14242} 
2016-05-10T08:01:41.214942Z [22648.101937089] user.info sam [] SAM GET_FOREGROUND_APPINFO {"CLOCK":22648.101,"PerfType":"AppLaunch","PerfGroup":"com.webos.app.discovery"} 
2016-05-10T08:01:41.253221Z [22648.140224797] user.info WebAppMgr [] WAM WINDOW_FOCUSIN {"CLOCK":22648.139,"PerfType":"AppLaunch","PerfGroup":"com.webos.app.discovery","APP_ID":"com.webos.app.discovery"}
