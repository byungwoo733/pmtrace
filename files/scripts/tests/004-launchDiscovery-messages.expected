Type: AppLaunch
Group: com.webos.app.discovery
Start time: 22644.60
Process                        MsgID                     Time(s)  +Diff(s)  Extra
surface-manager-starfish       IM_KEY_INPUT              0.0      +0.0      key_value:0 key_code:272
sam                            APP_LAUNCH                0.03     +0.03     status:start_prelaunching
sam                            APP_LAUNCH                0.043    +0.013    status:start_memory_checking
sam                            APP_LAUNCH                0.044    +0.001    status:start_launching
WebAppMgr                      APPLAUNCH_START           0.045    +0.001    APP_ID:com.webos.app.discovery
WebAppMgr                      APP_ATTACHED_TO_CONTAINER 0.047    +0.002    APP_ID:com.webos.app.discovery PID:14242
WebAppMgr                      APPLOADED                 3.403    +3.356    APP_ID:com.webos.app.discovery URL:https://kr.lgrecommends.lgappstv.com/discovery_2016/discovery.html?q=%7B%22returnValue%22%3Atrue%2C%22header%22%3A%22%22%2C%22HOST%22%3A%22KR.lgtvsdp.com%22%2C%22X-Device-Language%22%3A%22ko-KR%22%2C%22X-Device-FW-Version%22%3A%2200.00.00%22%2C%22X-Device- ... 3A%22HE_DTV_W16M_AFADATAA%22%2C%22X-Device-Eco-Info%22%3A%221%22%2C%22X-Device-Product%22%3A%22webOSTV+3.0%22%2C%22X-Device-Platform%22%3A%22W16M%22%2C%22X-Device-Publish-Flag%22%3A%22Y%22%2C%22X-Device-Sales-Model%22%3A%22WEBOS3%22%7D&page=store&fast=true PID:14242
sam                            GET_FOREGROUND_APPINFO    3.499    +0.096    
WebAppMgr                      WINDOW_FOCUSIN            3.537    +0.038    APP_ID:com.webos.app.discovery
Elapsed time (s) : 3.537

